src/ncp/ncp_base.cpp

```diff
@@ -76,6 +76,7 @@ namespace ot {
 #define NCP_INVALID_SCAN_CHANNEL              (-1)
 #define NCP_PLAT_RESET_REASON                 (1U<<31)
 #define NCP_ON_MESH_NETS_CHANGED_BIT_FLAG     (1U<<30)
+#define NCP_OFF_MESH_ROUTES_CHANGED_BIT_FlAG  (1U<<29)
```

Comment:
Good point. Will fix it. 

One other thing I'd like to change (maybe in separate PR) is to use a different variable for tracking NCP clanged flags separate from `mChangedFlags` (which is tracking changes coming from `HandleNetifStateChanged()`)